karma.conf.js

```diff
@@ -1,70 +1,88 @@
 // Karma configuration
 // Generated on Sun Jan 20 2019 23:06:22 GMT-0600 (CST)
 
+const isCI = !!process.env.CI;
```

Comment:
Does it throws error? I just found that I removed `progress` reporter from `karma`'s config, so it may look like it ends unexpectedly but without any error. I added it back, so it might be fixed.

Reply:
I replaced the `karma-safari-launcher` with the `karma-safarinative-launcher`. The original launcher is quite old and does not work correctly with the new versions of Safari (12+). The change should fix the error, could you check it please?